rsconf/package_data/sirepo_sandbox/sshd_config.jinja

```diff
@@ -0,0 +1,10 @@
+AuthorizedKeysFile {{ sirepo_sandbox.guest.identity_pub_f }}
+HostKey {{ sirepo_sandbox.guest.host_key_f }}
+ListenAddress {{ sirepo_sandbox.ip }}:{{ sirepo_sandbox.ssh_port }}
+PasswordAuthentication no
+PermitTunnel no
+PermitUserEnvironment no
```

Comment:
mpi_worker again, we are trying to control things so mpi works. for "normal" logins, this is fine.

Reply:
566ec33